plugins/flytekit-envd/flytekitplugins/envd/image_builder.py

```diff
@@ -88,7 +88,7 @@ def build():
     if image_spec.source_root:
         shutil.copytree(image_spec.source_root, pathlib.Path(cfg_path).parent, dirs_exist_ok=True)
         # Indentation is required by envd
-        envd_config += '    io.copy(host_path="./", envd_path="/root")'
+        envd_config += '    io.copy(source="./", target="/root")'
```

Comment:
it will break existing users. we should check `envd` version here. like

```python
if envd_version <= 0.3.36:
    envd_config += ' io.copy(host_path="./", envd_path="/root")'
else
    envd_config += ' io.copy(source="./", target="/root")'
```